src/explore-education-statistics-admin/src/pages/admin-dashboard/AdminDashboardPage.tsx

```diff
@@ -24,34 +26,40 @@ interface Model {
 const AdminDashboardPage = () => {
   const { user } = useAuthContext();
   const [model, setModel] = useState<Model>();
-  useEffect(() => {
-    Promise.all([
+  const reloadDashboard = useCallback(async () => {
```

Comment:
Can refactor the state, callback and effect to use the `useAsyncRetry` hook instead.

Reply:
Nice!  Have added that in.